package.json

```diff
@@ -37,26 +37,40 @@
     "url": "https://github.com/iTwin/insights-client"
   },
   "dependencies": {
-    "cross-fetch": "^3.1.4"
+    "@azure/storage-blob": "^12.11.0",
+    "@bentley/oidc-signin-tool": "^2.19.28",
+    "axios": "^0.27.2",
+    "chai-as-promised": "^7.1.1",
+    "chromium": "^3.0.3",
+    "copyfiles": "^2.4.1",
+    "cross-fetch": "^3.1.4",
```

Comment:
You should only need `cross-fetch` dependency. If tests require anything else, then it should be listed under `devDependencies`.

Reply:
moved all dependencies aside from cross-fetch